configs/odac/is2re/base.yml

```diff
@@ -0,0 +1,22 @@
+trainer: energy
+
+dataset:
+  train:
+    src: data_odac/is2r/train
+    normalize_labels: True
+    target_mean: -0.5249277949333191
+    target_std: 0.943034291267395
+  val:
+    src: data_odac/is2r/train
+
+logger:
+  name: wandb
+  project: odac_is2re
```

Comment:
Let's keep this general by removing the `project` specification?

Reply:
Done.